code/modules/mod/mod_actions.dm

```diff
@@ -2,28 +2,32 @@
 	background_icon_state = "bg_tech_blue"
 	icon_icon = 'icons/mob/actions/actions_mod.dmi'
 	check_flags = AB_CHECK_CONSCIOUS
+	/* SKYRAT EDIT REMOVAL - MODsuit pAIs
 	/// Whether this action is intended for the AI. Stuff breaks a lot if this is done differently.
 	var/ai_action = FALSE
+	*/ // SKYRAT EDIT END
```

Comment:
I really don't like the amount of non-modular changes to this file, it's gonna make major refactors a pain in the ass

Reply:
It was just so we wouldn't have a trailing variable that didn't do anything, but it's whatever I suppose. I'll be maintaining this nonetheless, though.